src/helpers/geoChart/data.ts

```diff
@@ -129,9 +129,10 @@ function getBucketItemNameAndDataIndex(
                     attributeHeader.uri === bucketItemInfo.uri,
             );
             if (index !== -1) {
+                const { formOf: { name = attributeHeaders[index].name } = {} } = attributeHeaders[index];
```

Comment:
It should be more clear as
```
const attributeName = attributeHeaders[index].name;
const { formOf: { name : attributeAlias } = {} } = attributeHeaders[index];
const name = attributeAlias || attributeName;
```

Reply:
Currently, the `formOf` always has in `attributeHeader` so I changed the code to:
```
const { formOf: { name } } = attributeHeaders[index];
result[bucketName] = { index, name };
```